src/main/java/com/iota/iri/service/API.java

```diff
@@ -996,7 +996,7 @@ private AbstractResponse getBalancesStatement(final List<String> addresses, fina
         if (tips == null || tips.size() == 0) {
             hashes = Collections.singletonList(instance.milestoneTracker.latestSolidSubtangleMilestone);
         } else {
-            hashes = tips.stream().map(address -> (HashFactory.ADDRESS.create(address)))
+            hashes = tips.stream().map(address -> (HashFactory.TRANSACTION.create(address)))
```

Comment:
Would it be an idea to use something different than **address**, eg. **tip** or **txhash**. Using **address** is confusing imo. Maybe i am completely off here but I thought let's just share it.

Reply:
@gjeee For cases like Milestones etc, this might be useful. But when you make certain transactions a `tip` object, you have to revert them again once they are being approved by others. (not specifically in this case, but for example storing transactions somewhere)

For now this PR just changes wrong types, and doesn't do more specific types

Reply:
this is not what I meant, but i see that your latest commit does what I was trying to suggest :-)

Reply:
Oooooh, i see what you mean now :)
Yes i noticed the strange param naming when you quoted it, so fixed it, thanks!